src/OrgUsageDashboard.vue

```diff
@@ -98,13 +99,25 @@ export default {
   },
   created() {
     this.isLoading = true;
+
+    const widgetId = Fliplet.Widget.getDefaultId();
+    const widgetData = Fliplet.Widget.getData(widgetId) || {};
+
+    this.featureAvailable = !!widgetData.featureAvailable;
   },
   mounted() {
-    const startDate = moment().add(-1, 'month');
-    const endDate = moment();
+    // If feature is available set correct dates
+    // If not, set to dates in the future to display no data
+    const startDate = this.featureAvailable
+      ? moment().add(-1, 'month')
+      : moment().add(1, 'month');
```

Comment:
Clever trick, but this will still send out a request to the server. Can we eliminate the request entirely?

Reply:
I tried that first, however we would have to build a sample data structure because the code expects to receive something to load up the UI correctly otherwise we will get a lot of console errors.

Do you recommend I put together a sample data and prevent the API request?

Reply:
@hcarneiro Yes please. I think it could also be a great way to demonstrate how the feature would look for those unable to access the feature.